notebooks/Basic simulation.ipynb

```diff
@@ -73,16 +84,20 @@
   }
  ],
  "metadata": {
+  "@webio": {
+   "lastCommId": "eb71d66f254443138c2b17f84b3959cb",
+   "lastKernelId": "0e644715-9dec-4de4-8c37-d946c1c45320"
+  },
   "kernelspec": {
-   "display_name": "Julia 1.0.0",
+   "display_name": "Julia 1.1.0",
    "language": "julia",
-   "name": "julia-1.0"
+   "name": "julia-1.1"
   },
   "language_info": {
    "file_extension": ".jl",
    "mimetype": "application/julia",
    "name": "julia",
-   "version": "1.0.0"
+   "version": "1.0.3"
```

Comment:
It's weird that this became 1.0.3 but the kernel is actually 1.1. I don't think it actually matters, though.